roles/container-engine/containerd/templates/containerd.service.j2

```diff
@@ -35,6 +35,9 @@ LimitNOFILE=infinity
 # Only systemd 226 and above support this version.
 TasksMax=infinity
 OOMScoreAdjust=-999
+{% if kube_reserved is defined and kube_reserved|bool %}
+Slice={{ kube_reserved_cgroups_for_service_slice }}
```

Comment:
Is there an equivalent for `cri-o` and `docker` (`cri-dockerd`) for these options? If not then the documentation should explicitly state that this capability is only available with `containerd`.

Reply:
I'm sorry, I misunderstood the yellow "Pending" decorator on my comments, so the above comment was just sent (it was actually commented many days ago).